library/src/animationManager.cxx

```diff
@@ -38,6 +38,18 @@ void animationManager::SetInteractor(interactor_impl* interactor)
   this->Interactor = interactor;
 }
 
+//----------------------------------------------------------------------------
+void animationManager::SetInteractorEventLoopTime(double loopTime)
+{
+  this->AnimationFrameNLoop = (1000.0 / this->Options.scene.animation.frame_rate) / loopTime;
```

Comment:
not a big fan of this but I dont have a better idea @Meakk

Reply:
I don't understand what you are trying to achieve.